Source/Core/VideoBackends/Vulkan/PostProcessing.cpp

```diff
@@ -143,7 +146,7 @@ static const std::string DEFAULT_FRAGMENT_SHADER_SOURCE = R"(
 
   void main()
   {
-    ocol0 = float4(texture(samp0, uv0).xyz, 1.0);
+    ocol0 = float4(texture(samp0, uv0));
```

Comment:
This may cause issues for frame dumping. That's the reason I fixed the alpha at 1 in the first place.

Reply:
What kind of issues di you see?

Reply:
If the video codec supports an alpha channel, it will use the values from the EFB/XFB/whatever instead of making every pixel opaque.